src/test/java/games/strategy/engine/lobby/client/ui/LobbyFrameTest.java

```diff
@@ -1,91 +0,0 @@
-package games.strategy.engine.lobby.client.ui;
-
-import static com.googlecode.catchexception.CatchException.catchException;
-import static com.googlecode.catchexception.CatchException.caughtException;
-import static com.googlecode.catchexception.apis.CatchExceptionHamcrestMatchers.hasMessageThat;
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.instanceOf;
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
-
-import java.time.Instant;
-
-import org.junit.Test;
-
-import games.strategy.engine.lobby.client.ui.LobbyFrame.TimeUnitNames;
-
-public final class LobbyFrameTest {
```

Comment:
A little background on these tests...  They were added to cover the very subtle `UnsupportedTemporalTypeException` that can occur at runtime when using non-constant periodic temporal types like `ChronoUnit#MONTHS`.  That's why we have to use `LocalDateTime` as an intermediate type for those types of units.  These tests were there to prevent regressions in the event someone came along and changed the implementation.  It seems like that could still be possible with the `TimespanDialog$TimeUnit` implementations.